code/modules/projectiles/projectile/bullets/shotgun.dm

```diff
@@ -1,11 +1,11 @@
 /obj/projectile/bullet/shotgun_slug
 	name = "12g shotgun slug"
-	damage = 60
+	damage = 45

```

Comment:
the pr says that you cahnged it to 40 but its 45

Reply:
whoops